ui/mirage/factories/metrics/activity.js

```diff
@@ -6,8 +6,8 @@ export default Mirage.Factory.extend({
   total: function () {
     return {
       clients: 20,
-      distinct_entities: 10,
-      non_entity_tokens: 10,
+      entity_clients: 10,
```

Comment:
I updated these key names, but I have no idea what this file is for..

Reply:
Let's not modify this file as we're not using it. However, let's add a note that it may be old code and could be removed. Maybe make a ticket for it. e.g. clean up work.

Reply:
The metrics page has been redone several times so it's likely it's leftover, but needs investigation.